asynchronous_web_services/async_http/oauth.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python2.7
+
+import time
+import oauth2
+
+CONSUMER_KEY = ''
+CONSUMER_SECRET = ''
+ACCESS_KEY = ''
+ACCESS_SECRET = ''
+
+
+def oauth(url):
+
+	CONSUMER = oauth2.Consumer(CONSUMER_KEY, CONSUMER_SECRET)
+	access_token = oauth2.Token(key=ACCESS_KEY, secret=ACCESS_SECRET)
+
+	params = {
+		'oauth_version': "1.0",
+		'oauth_nonce': oauth2.generate_nonce(),
+		'oauth_timestamp': int(time.time()),
+		'oauth_token': access_token.key,
+		'oauth_consumer_key': CONSUMER.key
```

Comment:
@grubberr :+1: 

We can also add,

`'oauth_signature_method': "HMAC-SHA1"`

But will this have any effect on Line #28 (`req.sign_request()`) ?